### DISCUSSION TITLE
Github page not displaying properly, please help

### DISCUSSION THREAD

**Discussion by archetypebiomedical** [community/community#22771] [Other Feature Feedback, Questions, & Ideas]
<p>Greetings,</p>
<p>I am trying to setup a github page using this jekyll theme <a href="https://github.com/melvinchng/business-jekyll-theme" rel="noopener nofollow ugc">https://github.com/melvinchng/business-jekyll-theme</a> which itself is hosted on github.</p>
<p>I downloaded the zip file, created a local git repository, and made sure the web page can be developed locally. Upon first execution of:</p>
<pre><code class="lang-auto">bundle exec jekyll serve
</code></pre>
<p>on my local machine, I got the error message of:</p>
<pre><code class="lang-auto">Bundler could not find compatible versions for gem "ffi":
  In snapshot (Gemfile.lock):
    ffi (= 1.9.23)

  In Gemfile:
    jekyll (= 3.5.2) was resolved to 3.5.2, which depends on
      jekyll-watch (~&gt; 1.1) was resolved to 1.5.1, which depends on
        listen (~&gt; 3.0) was resolved to 3.1.5, which depends on
          rb-inotify (&gt;= 0.9.7, ~&gt; 0.9) was resolved to 0.9.10, which depends on
            ffi (&lt; 2, &gt;= 0.5.0)

Running `bundle update` will rebuild your snapshot from scratch, using only
the gems in your Gemfile, which may resolve the conflict.
</code></pre>
<p>Thus I had to update it using <code>bundle update</code>. After that, the webpage can be generated and displayed correctly on my local machine.</p>
<p>I then synced my local repository to my github page at:</p>
<p><a href="https://archetype-biomedical.github.io/Archetype-Biomedical/" rel="noopener nofollow ugc">https://archetype-biomedical.github.io/Archetype-Biomedical/</a></p>
<p>but as you can see, the webpages are not being generated/displayed properly as compared to the original jekyll theme that is also hosted on the github.</p>
<p>Can anyone please shed some lights as what I am doing wrong?  I have hosted personal github page using other jekyll theme and had worked well. I must admit that I have not used this particular theme previously until now.</p>
<p>any help is very much appreciated,</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/archetypebiomedical">@archetypebiomedical</a>, welcome to the GitHub Support Community!</p>
<p>Taking a look in the console for your site it seems like like your assets are 404-ing due to incorrect URLs. From your source code, your assets are being loaded using using a relative link:</p>
<pre><code class="lang-auto">&lt;link rel="stylesheet" href="../assets/css/style.css"&gt;
</code></pre>
<p>This link will resolve to the following URL:</p>
<pre><code class="lang-auto">https://archetype-biomedical.github.io/assets/css/style.css
</code></pre>
<p>This is happening because of the relative path traversal <code>../</code> at the start of these asset links, causing the browser to traverse up one directory level in your URL.</p>
<p>This CSS file is actually located at the following URL:</p>
<pre><code class="lang-auto">https://archetype-biomedical.github.io/Archetype-Biomedical/assets/css/style.css
</code></pre>
<p>Notice the extra <code>Archetype-Biomedical</code> section in there—the name of the repository! The browser reads this part of the URL as a directory, meaning the <code>../</code> will move “up one level,” out of this “directory,” and remove this section of your asset URLs.</p>
<p>The best way to fix this would be to update all your asset links to include this part. You could simply replace <code>../</code> with <code>/Archetype-Biomedical/</code>, but if you ever changed the name of your repository then these would break again.</p>
<p>The best thing to do would be to use Jekyll to automatically include the <code>baseurl</code> value—your repository name—in these links.</p>
<p>To do this you’ll need to update all your asset links to prepend the <code>baseurl</code> value. For example, using the CSS file above, you’ll need to change your link to this:</p>
<pre><code class="lang-auto">&lt;link rel="stylesheet" href="{{ "/assets/css/style.css" | relative_url }}"&gt;
</code></pre>
<p>GitHub <em>should</em> set this <code>baseurl</code> value automatically but as it’s explicitly defined in your repository you may also need to set your <code>baseurl</code> value to the name of your repository inside your <code>_config.yml</code> file:</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/Archetype-Biomedical/Archetype-Biomedical/blob/683b3ea0b50f49e576b8368d586106f2ebc1a612/_config.yml#L21" target="_blank" rel="noopener">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/Archetype-Biomedical/Archetype-Biomedical/blob/683b3ea0b50f49e576b8368d586106f2ebc1a612/_config.yml#L21" target="_blank" rel="noopener">Archetype-Biomedical/Archetype-Biomedical/blob/683b3ea0b50f49e576b8368d586106f2ebc1a612/_config.yml#L21</a></h4>
<pre class="onebox"><code class="lang-yml"><ol class="start lines" start="11" style="counter-reset: li-counter 10 ;">
<li># Site settings</li>
<li># These are used to personalize your new site. If you look in the HTML files,</li>
<li># you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.</li>
<li># You can create any custom variable you would like, and they will be accessible</li>
<li># in the templates via {{ site.myvariable }}.</li>
<li>title: Business Jekyll Theme</li>
<li>email: your-email@example.com</li>
<li>description: &gt; # this means to ignore newlines until "baseurl:"</li>
<li>  Yet another Jekyll theme. It is designed for business purpose and ported by </li>
<li>  Melvin Ch'ng.</li>
<li class="selected">baseurl: "" # the subpath of your site, e.g. /blog</li>
<li>url: "" # the base hostname &amp; protocol for your site, e.g. http://example.com</li>
<li>
</li>
<li># Company Information</li>
<li>company_name: Johns, Smith &amp; Associates</li>
<li>address_line: Via Ludovisi 39-45, Rome, 54267, Italy</li>
<li>telephone_number: 1.800.245.356</li>
<li>fax_number: 1.800.245.357</li>
<li>company_email: hello@LawOffice.org</li>
<li>google_map: https://www.google.com/maps/embed?pb=!1m14!1m12!1m3!1d3618.727010735933!2d91.837871!3d24.907291700000002!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!5e0!3m2!1sen!2s!4v1404919738144</li>
<li>
</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>You can read about the <code>relative_url</code> filter, and other useful Jekyll filters, here:<br>
</p><aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://jekyllrb.com/favicon.ico" class="site-icon" width="32" height="32">
      <a href="https://jekyllrb.com/docs/liquid/filters/" target="_blank" rel="noopener">Jekyll • Simple, blog-aware, static sites</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/2439407/181073084-6a644c10-0be7-4613-8783-a938960920c6.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_10x10.png"></div>

<h3><a href="https://jekyllrb.com/docs/liquid/filters/" target="_blank" rel="noopener">Liquid Filters</a></h3>

<p>All of the standard Liquid filters are supported (see below).</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Reply by Bill-Adepoju** [reply]
Hi, can someone help me check out this also? My css is not showing in the pages. https://github.com/Bill-Adepoju/3-preview

https://bill-adepoju.github.io/3-preview/
